pkg/api/capability/capability.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package capability
+
+var watchCapability map[string]int
+
+func init() {
+	watchCapability = make(map[string]int)
+	watchCapability["pods"] = 1000
```

Comment:
I recommend finding constants to use here, they should be buried somewhere in the various registry packages?